jobs/resources-import-consumer/index.ts

```diff
@@ -56,7 +54,7 @@ const upsertRecord = async (message: ImportRequestBody): Promise<void> => {
 };
 
 type ProcessedResult = {
-  status: 'successs';
+  status: 'success';
```

Comment:
Ha thanks for this one, it would bother me too if I knew I was speaking [pársel](https://harrypotter.fandom.com/es/wiki/P%C3%A1rsel) xD